packages/remix-react/components.tsx

```diff
@@ -1178,17 +1168,17 @@ export function useActionData<T = AppData>(): SerializeFrom<T> | undefined {
  */
 export function useFetchers(): Fetcher[] {
   let fetchers = useFetchersRR();
-  return fetchers.map((f) =>
-    convertRouterFetcherToRemixFetcher({
+  return fetchers.map((f) => {
+    let fetcher = convertRouterFetcherToRemixFetcher({
```

Comment:
It looks to me like the interface for `fetchers` has all these props and nothing else. If so then you could just return this

```TS
return fetchers.map(convertRouterFetcherToRemixFetcher);
```

Reply:
Good catch!